test/unit/org/apache/cassandra/index/sai/SAIUtil.java

```diff
@@ -97,6 +100,25 @@ public static void setCurrentVersion(Version.Selector versionSelector)
         }
     }
 
+    public static void setEnableNVQ(boolean enableNVQ)
+    {
+        try
+        {
+            CassandraRelevantProperties.SAI_VECTOR_ENABLE_NVQ.setBoolean(enableNVQ);
+            Field field = NVQUtil.class.getDeclaredField("ENABLE_NVQ");
```

Comment:
Not related to this patch:
I don't know why we don't have an utility method to set values to private fields ? I have seen this code lot of times